nestor_api/config/k8s.py

```diff
@@ -2,11 +2,15 @@
 import os
 
 
-# pylint: disable=too-few-public-methods
 class K8sConfiguration:
     """k8s configuration"""
 
     @staticmethod
     def get_http_proxy() -> str:
         """Returns the k8s http proxy"""
         return os.environ["NESTOR_K8S_HTTP_PROXY"]
+
+    @staticmethod
+    def get_templates_dir() -> str:
+        """Returns the name of the config sub-folder in which to look for templates."""
+        return os.getenv("NESTOR_K8S_TEMPLATE_FOLDER", "templates")
```

Comment:
IMO this can be resolved using NESTOR_CONFIG_PATH concatenated with `templates`

Reply:
I plan to use it this way :
```python
from nestor_api.config.k8s import K8sConfiguration
import nestor_api.lib.config as config

config_dir = config.create_temporary_config_copy()
templates_path = os.path.join(config_dir, K8sConfiguration.get_templates_dir())
```

Reply:
(This makes me realize that I forgot to reference the variable in the README)